### PR TITLE
Fix action generation for app architecture

### PR DESCRIPTION
**Pull request by davydovanton**
/cc @jodosha

### CODE REVIEW

**Comment by davydovanton** [comment]
I have failed test with msql adapter :disappointed: 
```
Lotus::Commands::New::App::rspec#test_0001_creates files [/home/travis/build/lotus/lotus/test/commands/new/app_test.rb:54]:
--- expected
+++ actual
@@ -12,7 +12,7 @@
   #  * SQL adapter
   #    adapter type: :sql, uri: 'sqlite://db/new_app_development.sqlite3'
   #    adapter type: :sql, uri: 'postgres://localhost/new_app_development'
-  #    adapter type: :sql, uri: 'mysql://localhost/new_app_development'
+  #    adapter type: :sql, uri: 'mysql2://localhost/new_app_development'
   #
   adapter type: :file_system, uri: ENV['NEW_APP_DATABASE_URL']
   ##
```

**Comment by jodosha** [comment]
@davydovanton Thank you for being so quick and responsive :+1: 
I've reverted the commit that caused the failure here. Can you please merge `master` here and push again? That should fix the build for this PR. TY!

File: lib/lotus/cli_sub_commands/generate.rb [hunk 1]
```diff
@@ -36,7 +36,13 @@ class Generate < Thor
       method_option :test, desc: 'Defines the testing Framework to be used. Default is defined through your .lotusrc file.'
       method_option :skip_view, desc: 'Skip the generation of the view. Also skips template generation.', default: false, type: :boolean
       method_option :template, desc: 'Extension to be used for the generated template. Default is defined through your .lotusrc file.'
-      def actions(application_name, controller_and_action_name)
+      def actions(application_name = nil, controller_and_action_name)
+        if Lotus::Environment.new(options).container? && application_name.nil?
+          msg = "ERROR: \"lotus actions\" was called with arguments [\"#{controller_and_action_name}\"]\n" \
```
Comment (jodosha) [inline lib/lotus/cli_sub_commands/generate.rb]: @davydovanton It's `action`, not `actions` :wink:

**Comment by jodosha** [comment]
@davydovanton As last thing, we need a test to ensure this won't happen again in the future. Can you please write it? Thanks.

**Comment by davydovanton** [comment]
@jodosha done. I added tests :tada:

**Comment by joneslee85** [comment]
:+1:

**Comment by davydovanton** [comment]
@jodosha what do you think? Can you merge this PR?

**Comment by jodosha** [comment]
@davydovanton Yes, it is. Thank you very much! :+1:

**Comment by davydovanton** [comment]
thanks!

**Pull request merged**